db/migrate/20200114203042_add_missing_platform_info.rb

```diff
@@ -5,6 +5,16 @@ def up
     runs.each do |run_id|
       run = Run.find(run_id)
       run.update_platform_info(run.sequence_run.attributes)
+      # Sending run to firestore
+      # check that the local environment has
+      # ENV["REPORT_SERVICE_SELF_URL"]
+      # ENV["REPORT_SERVICE_URL"]
+      # ENV["REPORT_SERVICE_TOKEN"]
+      sender = ReportService::RunSender.new(run, { send_all_answers: true })
+      result = sender.send
+      if !result || !result["success"]
+        puts "Failed to send Sequence Run: #{run.id}"
```

Comment:
Since this is now sending a activity run instead of a sequence run, this log message should be updated.
I'm assuming the RunSender will work fine with either sequence or activities runs, right?

Reply:
Yes, your code is also sending activity runs, not sequence runs. I changed this log message by accident. Fixed, thanks.